### PR TITLE
README: add Sealos 1-click deployment

### PR DESCRIPTION
**Pull request by yangchuansheng**

### CODE REVIEW

**Review by thijsvanloef — CHANGES_REQUESTED** [review]
Hi @yangchuansheng I am not familiar with SealOS, could you please provide some more information about it in the readme?

**Comment by yangchuansheng** [comment]
> Hi @yangchuansheng I am not familiar with SealOS, could you please provide some more information about it in the readme?

[Sealos](https://github.com/labring/sealos) is also a well-known open-source project that can start all dependencies of an application, including the database, with just one click. It is very convenient. You can also try clicking the one-click installation button to get palworld server up and running in a few minutes. Sealos currently has nearly 100,000 registered users and is one of the largest container hosting platforms in China.

**Comment by anymouschina** [comment]
> > 你好@yangchuansheng我对 SealOS 不熟悉，您能否在自述文件中提供更多有关它的信息？
> 
> [Sealos](https://github.com/labring/sealos)也是一个著名的开源项目，只需一键即可启动应用程序的所有依赖项，包括数据库。非常方便。您还可以尝试单击一键安装按钮，在几分钟内启动并运行 palworld 服务器。Sealos目前拥有近10万注册用户，是国内最大的容器托管平台之一。

你好，seaIos 模板市场部署后如何连接呢？我找不到公网ip，通过ping外网url拿到的ip，在游戏内搜不到该服务器，希望模板 readme 内能将后续步骤补充至游戏进入服务器

**Comment by yangchuansheng** [comment]
> > > 你好@yangchuansheng我对 SealOS 不熟悉，您能否在自述文件中提供更多有关它的信息？
> > 
> > 
> > [Sealos](https://github.com/labring/sealos)也是一个著名的开源项目，只需一键即可启动应用程序的所有依赖项，包括数据库。非常方便。您还可以尝试单击一键安装按钮，在几分钟内启动并运行 palworld 服务器。Sealos目前拥有近10万注册用户，是国内最大的容器托管平台之一。
> 
> 你好，seaIos 模板市场部署后如何连接呢？我找不到公网ip，通过ping外网url拿到的ip，在游戏内搜不到该服务器，希望模板 readme 内能将后续步骤补充至游戏进入服务器

这里有一篇文章详细描述了部署和连接过程：https://www.cnblogs.com/ryanyangcs/p/17984858

**Review by Twinki14 — COMMENTED** [review]
My only nit-pick with this is that it's adding what's effectively an ad to the readme

Instead, could a basic hyper link to the SealOS template be added under the Kubernetes section?

File: README.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ docker run -d \
 
 ```
 
+### Deploy on Sealos
+
+[Sealos](https://github.com/labring/sealos) is a cloud operating system distribution based on the Kubernetes kernel. It enables the initialization of all application dependencies, including the database, with a single click.
+
+[![](https://raw.githubusercontent.com/labring-actions/templates/main/Deploy-on-Sealos.svg)](https://template.cloud.sealos.io/deploy?templateName=palworld)
+
```
Comment (Twinki14) [inline README.md:81]: My only nit-pick with this is that it's adding what's effectively an ad to the readme

Instead, could a basic hyper link to the SealOS template be added under the Kubernetes section?

**Comment by thijsvanloef** [comment]
I've given this PR some thought. I've decided against implementing this change. Like @Twinki14 mentioned, this feels like adding an add to the repository, while adding little (or no) value. 
I do want to thank you for taking the time to open this PR.

**Pull request closed without merging**